### PR TITLE
feat: add telegram & gitter social links

### LINKED ISSUE #5022: Add Gitter and Telegram to footer and admin pages
Add Telegram and Gitter.im to footer and admin pages here https://eventyay.com/admin/content

Screenshot from 2020-09-09 08-56-38

### PR DESCRIPTION
**Pull request by progmatic-99**
Requires this [PR](https://github.com/fossasia/open-event-server/pull/7266)

Fixes #5022

### CODE REVIEW

File: app/templates/components/forms/admin/content/social-links-form.hbs [hunk 1]
```diff
@@ -61,5 +61,21 @@
       @isSocialLink={{true}}
       @inputId="patreon" />
   </div>
+  <div class="field">
+    <label>{{t 'Gitter'}}</label>
+    <Widgets::Forms::LinkInput
+      @segmentedLink={{this.socials.segmentedGitterUrl}}
```
Comment (iamareebjamal) [inline app/templates/components/forms/admin/content/social-links-form.hbs:67]: This does not exist

File: app/templates/components/forms/admin/content/social-links-form.hbs [hunk 2]
```diff
@@ -61,5 +61,21 @@
...
+      @linkName="gitter"
+      @isSocialLink={{true}}
+      @inputId="gitter" />
+  </div>
+  <div class="field">
+    <label>{{t 'Telegram'}}</label>
+    <Widgets::Forms::LinkInput
+      @segmentedLink={{this.socials.segmentedTelegramUrl}}
```
Comment (iamareebjamal) [inline app/templates/components/forms/admin/content/social-links-form.hbs:75]: This does not exist

File: app/templates/components/forms/admin/content/social-links-form.hbs (hunk 1 above)
Comment (progmatic-99) [reply app/templates/components/forms/admin/content/social-links-form.hbs:67]: What do you mean?
Is it not appearing on the front-end ?

Comment (iamareebjamal) [reply app/templates/components/forms/admin/content/social-links-form.hbs:67]: segmentedGitterUrl does not exist

File: app/models/setting.js [hunk 3]
```diff
@@ -108,6 +108,8 @@ export default ModelBase.extend({
   segmentedFacebookUrl   : computedSegmentedLink.bind(this)('facebookUrl'),
   segmentedInstagramUrl  : computedSegmentedLink.bind(this)('instagramUrl'),
   segmentedPatreonUrl    : computedSegmentedLink.bind(this)('patreonUrl'),
+  segmentedGitterUrl     : computedSegmentedLink.bind(this)('gitterUrl'),
+  segmentedTelegramUrl   : computedSegmentedLink.bind(this)('telegramUrl'),
```
Comment (iamareebjamal) [inline app/models/setting.js:112]: Add them below

**Review by iamareebjamal — APPROVED** [review]

**Comment by mariobehling** [comment]
Thanks. There is still a small issue though. As soon as I enter the link "https://" switches to "https". So, the links do not work due to missing "://".

**Pull request merged**